lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -377,6 +377,107 @@ class DecomposeAtenZeroOp
 };
 } // namespace
 
+namespace {
+class DecomposeAtenEyeOp
+    : public OpRewritePattern<AtenEyeOp> {
+public:
+  using OpRewritePattern::OpRewritePattern;
+  LogicalResult matchAndRewrite(AtenEyeOp op,
+                                PatternRewriter &rewriter) const override {
+    Value n = op.getN();
+    Value m = op.getN();
+    rewriter.replaceOpWithNewOp<AtenEyeMOp>(op, op.getType(), n, m, op.getDtype(), op.getLayout(), op.getDevice(), op.getPinMemory());
```

Comment:
nit: the formatting is off. Make sure you use `git clang-format` to format your changes